index.html

```diff
@@ -25,12 +26,31 @@
         name="keywords" 
         content="" 
     />
+
+    <link 
+        rel="stylesheet"
+        href="css/style.css"
+        type="text/css"
+    />
+
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js">
```

Comment:
Once you've removed the jQuery usage, please remove the library and file import as well.